model/riscv_sys_regs.sail

```diff
@@ -294,6 +255,67 @@ function set_mstatus_UXL(m : Mstatus, a : arch_xlen) -> Mstatus = {
   }
 }
 
+val extension : (string) -> bool
+
+function extension(ext) = {
+  match ext {
+    /* helpers to check support for various extensions. */
+    /* we currently don't model 'E', so always assume 'I'. */
```

Comment:
Move these 2 lines above `function extension` line, above.
Also, change "helpers" to singular "helper".